src/components/Events/Events.vue

```diff
@@ -1,6 +1,22 @@
 <template>
     <div>
-        <h1>Events</h1>
-        <img src="../../../assets/events.png" width="1200" height="800">
+        <h1>Our Upcoming Events</h1>
+        <TheEventsList />
     </div>
 </template>
+
```

Comment:
I think that it is better to move TheEventList component just into this events.vue file here. Especially since we're going to be using it in multiple now.